boards/bytesatwork/bytesensi_l/bytesensi_l.dts

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyrigtt (c) 2024 bytesatwork AG
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nordic/nrf52832_qfaa.dtsi>
+#include "bytesensi_l_pinctrl.dtsi"
+
+#include <zephyr/dt-bindings/gpio/gpio.h>
+
+/ {
+	model = "bytesatwork BLE/LORA sensor board";
+	compatible = "bytesatwork,bytesensi-l";
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+	};
+
+	zephyr,user {
+		enable-1wire-gpios = <&gpio0 7 GPIO_ACTIVE_HIGH>;
+	};
```

Comment:
This is no longer needed.

Reply:
Yes, I thought about that during the lunch :see_no_evil: , good catch!